src/main/kotlin/com/autonomousapps/internal/advice/ProjectHealthConsoleReportBuilder.kt

```diff
@@ -48,7 +48,8 @@ internal class ProjectHealthConsoleReportBuilder(
 
       if (addAdvice.isNotEmpty()) {
         maybeAppendTwoLines()
-        appendReproducibleNewLine("Transitively used dependencies that should be declared directly as indicated:")
+        appendReproducibleNewLine("This project requires all transitive dependencies to be declared directly. " +
+          "Add these to your build.gradle:")
```

Comment:
What about simply:
```
These transitive dependencies should be declared directly:
```

Reply:
I think that's a good step.  Should it include the path the the build file or is that obvious from context?

Reply:
No other advice includes a path to the build file, and in fact that path is technically dynamic so you shouldn't hardcode it. So, I'd rather not mention it at all. I will assume for now that people know where to declare their dependencies.

Reply:
updated